test/e2e/node/node_problem_detector.go

```diff
@@ -147,11 +147,11 @@ var _ = SIGDescribe("NodeProblemDetector [DisabledForLargeClusters]", func() {
 		for i := 1; i <= numIterations; i++ {
 			for j, host := range hosts {
 				if isStandaloneMode[host] {
-					rss, workingSet := getMemoryStat(f, host)
+					rss, workingSet := getMemoryStat(host)
 					rssStats[host] = append(rssStats[host], rss)
 					workingSetStats[host] = append(workingSetStats[host], workingSet)
 					if i == numIterations {
-						cpuUsage, uptime := getCPUStat(f, host)
+						cpuUsage, uptime := getCPUStat(host)
 						cpuUsageStats[host] = append(cpuUsageStats[host], cpuUsage)
 						uptimeStats[host] = append(uptimeStats[host], uptime)
 					}
```

Comment:
For instance there's this particular code snippet :
https://github.com/kubernetes/kubernetes/blob/c5cc25d1cb075f84756db9bffecd3fafe12c5fab/test/e2e/node/node_problem_detector.go#L215